tests/Tests/500__jupyterhub/test-filling-pvc.robot

```diff
@@ -0,0 +1,40 @@
+*** Settings ***
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library             DebugLibrary
+
+Suite Setup         Server Setup
+Suite Teardown      End Web Test
+
+
+*** Variables ***
+${PATH_TO_FILE}         ods-ci-notebooks-main/notebooks/200__monitor_and_manage/203__alerts/notebook-pvc-usage/fill-notebook-pvc-to-complete-100.ipynb
+${LINK_OF_GITHUB}       https://github.com/redhat-rhods-qe/ods-ci-notebooks-main.git
+${FILE_NAME}            fill-notebook-pvc-to-complete-100.ipynb
+
+
+*** Test Cases ***
+Get Notifications If Storage Capacity Limits Get Exceeded
+    [Documentation]    It clones the ${LINK_OF_GITHUB} repo and runs ${FILE_NAME}, then it checks for the OS Error
+    ...    which is generated by running that file and also checks for the popup which shows File Save Error
+    [Tags]    Sanity    ODS-539
+
+    Run Keyword And Ignore Error    Clone Git Repository And Run    ${LINK_OF_GITHUB}    ${PATH_TO_FILE}
+
+    ${output} =    Get Text    (//div[contains(@class,"jp-OutputArea-output")])[last()]
+    ${output} =    Split String    ${output}    \n\n
```

Comment:
I would put lines 24,26 and 27 in a dedicate keyword.
Then, you could try using an already defined keyword to get the error test in JL: `Get JupyterLab Code Cell Error Text`.

Reply:
I have seen that [keyword ](https://github.com/red-hat-data-services/ods-ci/blob/c8b5032df13ea04a2541d0c36c2a3161682c11f8/tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot#L67)it has return keyword in it but it is not returning the error

Reply:
hm try to change it to return the error text. It might be an error in the keyword since the [Return] is empty.
However, if it doesn't work I think it's fine what you did. Wrap lines 24-26-27 in a keyword to make the TC more readable

Reply:
Yep, looks like that keyword is broken and missing the return parameter. 
I agree with Berto's idea of wrapping these three lines in a keyword, other than that LGTM